app/projects/validations.ts

```diff
@@ -61,7 +61,7 @@ export const FullFormFields = {
   repoUrls: z
     .array(
       z.object({
-        id: z.string().optional(),
+        id: z.string().optional().or(z.number()),
```

Comment:
why not just z.number()?

Reply:
I did it that way because I don't know if there is any possibility for this Id to change in the future for a String or something, so I decided to leave it validating both options, but I'll change it then to leave it as a number value only.

Thanks for the feedback!